### PR TITLE
.

### PR DESCRIPTION
**Pull request by alejoard6767**
tank man assest

### CODE REVIEW

File: tankman_assets [hunk 1]
```diff
@@ -0,0 +1 @@
+https://www.mediafire.com/file/xqqydl8vyuzlnwg/Tankman_assets.fla/file
```
Comment (alejoard6767) [inline tankman_assets:1]: this is the tank man assets

**Comment by TentaRJ** [comment]
Why do you want to add this?

**Comment by JJ-GRAVITY** [comment]
> Why do you want to add this?

Maybe it's almost time to release week 7.

**Comment by CaptainGame17** [comment]
this pr shouldnt exist

**Comment by Riteo** [comment]
what

**Comment by The-SGPT** [comment]
this shouldn't even exist cos an fla is already made, just not pushed yet

**Comment by exurd** [comment]
works on flash professional cs5.5 so it's worthy for a merge-y ;))

tankmansworld

**Comment by alejoard6767** [comment]
tanks to everyone what see the file (you get it?)

File: tankman_assets (hunk 1 above)
Comment (alejoard6767) [reply tankman_assets:1]: tanks to everyone  who see the file (you get it?)

**Comment by alejoard6767** [comment]
@exurd  thanks for send a screenshot now i know that work´s

**Comment by exurd** [comment]
> @exurd thanks for send a screenshot now i know that work´s

/j

**Comment by alejoard6767** [comment]
if you do an animation of that please give credit but noting more (sorry for bad english)

**Comment by alejoard6767** [comment]
@TentaRJ  im waiting for it

File: tankman_assets (hunk 1 above)
Comment (alejoard6767) [reply tankman_assets:1]: ATENTION IM GONNA TO UPDATE THE FILE TO PUT THE ICONS OF TANKMAN

**Comment by TentaRJ** [comment]
> @TentaRJ im waiting for it

Good. Wait longer without pinging me next time, alright?

File: tankman_assets (hunk 1 above)
Comment (alejoard6767) [inline tankman_assets:1]: https://www.mediafire.com/file/ewhqot044idj8h7/Tankman_assets.fla/file New tankman files

**Comment by alejoard6767** [comment]
ok

**Comment by alejoard6767** [comment]
@exurd  you can put more screenshots? please just to see if work´s good

**Comment by alejoard6767** [comment]
why no one is chating??

**Comment by CaptainGame17** [comment]
> why no one is chating??

this isnt discord lol

**Comment by alejoard6767** [comment]
oh sorry

**Comment by alejoard6767** [comment]
please someone can send images of the file? just to see if work

**Review by TentaRJ — CHANGES_REQUESTED** [review]
Don't push this. It doesn't need to be pushed. Please close the request.

File: tankman_assets (hunk 1 above)
[the remark this answers is not in the document]
Comment (alejoard6767) [reply tankman_assets:1]: i cant sent it directly

**Comment by hdboye** [comment]
please close this pr. pull requests here are for fixes or improvements to the code. not to add assets which are still being worked on

**Comment by alejoard6767** [comment]
im not gonna to close the recuest i worked on the file and in type in english im not english i talk spanish

**Comment by alejoard6767** [comment]
and i dont know how to close it

**Comment by hdboye** [comment]
the fnf team is working on week 7 themselves, they have the assets already. all of these files will eventually be uploaded to the github once week 7 releases.

**Comment by hdboye** [comment]
> and i dont know how to close it

to the left of the comment button, click Close Pull Request

**Comment by hdboye** [comment]
thank you

**Comment by alejoard6767** [comment]
sorry

**Pull request closed without merging**